<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ru.stqa.pft.mantis.tests.ChangePassword" tests="2" skipped="1" failures="1" errors="0" timestamp="2020-06-12T13:12:18" hostname="LAPTOP-9JV1292F" time="0.116">
  <properties/>
  <testcase name="startMailServer" classname="ru.stqa.pft.mantis.tests.ChangePassword" time="0.101">
    <failure message="java.lang.RuntimeException: java.net.BindException: Address already in use: JVM_Bind" type="java.lang.RuntimeException">java.lang.RuntimeException: java.net.BindException: Address already in use: JVM_Bind
	at org.subethamail.smtp.server.SMTPServer.start(SMTPServer.java:292)
	at org.subethamail.wiser.Wiser.start(Wiser.java:90)
	at ru.stqa.pft.mantis.appmanager.MailHelper.start(MailHelper.java:50)
	at ru.stqa.pft.mantis.tests.ChangePassword.startMailServer(ChangePassword.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:63)
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:348)
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:695)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:523)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.testng.TestRunner.privateRun(TestRunner.java:766)
	at org.testng.TestRunner.run(TestRunner.java:587)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
	at org.testng.SuiteRunner.run(SuiteRunner.java:286)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
	at org.testng.TestNG.runSuites(TestNG.java:1039)
	at org.testng.TestNG.run(TestNG.java:1007)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.runTests(TestNGTestClassProcessor.java:141)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.stop(TestNGTestClassProcessor.java:90)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.stop(SuiteTestClassProcessor.java:61)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at com.sun.proxy.$Proxy2.stop(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.stop(TestWorker.java:132)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:182)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:164)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:412)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.BindException: Address already in use: JVM_Bind
	at java.net.DualStackPlainSocketImpl.bind0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketBind(DualStackPlainSocketImpl.java:106)
	at java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:387)
	at java.net.PlainSocketImpl.bind(PlainSocketImpl.java:190)
	at java.net.ServerSocket.bind(ServerSocket.java:375)
	at org.subethamail.smtp.server.SMTPServer.createServerSocket(SMTPServer.java:334)
	at org.subethamail.smtp.server.SMTPServer.start(SMTPServer.java:288)
	... 60 more
</failure>
  </testcase>
  <testcase name="testChangePassword" classname="ru.stqa.pft.mantis.tests.ChangePassword" time="0.0">
    <skipped/>
  </testcase>
  <system-out><![CDATA[16:12:18.922 [Test worker] DEBUG o.s.smtp.server.CommandHandler - Added command: AUTH
16:12:18.922 [Test worker] DEBUG o.s.smtp.server.CommandHandler - Added command: DATA
16:12:18.922 [Test worker] DEBUG o.s.smtp.server.CommandHandler - Added command: EHLO
16:12:18.922 [Test worker] DEBUG o.s.smtp.server.CommandHandler - Added command: HELO
16:12:18.922 [Test worker] DEBUG o.s.smtp.server.CommandHandler - Added command: HELP
16:12:18.922 [Test worker] DEBUG o.s.smtp.server.CommandHandler - Added command: MAIL
16:12:18.922 [Test worker] DEBUG o.s.smtp.server.CommandHandler - Added command: NOOP
16:12:18.922 [Test worker] DEBUG o.s.smtp.server.CommandHandler - Added command: QUIT
16:12:18.922 [Test worker] DEBUG o.s.smtp.server.CommandHandler - Added command: RCPT
16:12:18.922 [Test worker] DEBUG o.s.smtp.server.CommandHandler - Added command: RSET
16:12:18.922 [Test worker] DEBUG o.s.smtp.server.CommandHandler - Added command: STARTTLS
16:12:18.922 [Test worker] DEBUG o.s.smtp.server.CommandHandler - Added command: VRFY
16:12:18.922 [Test worker] INFO  o.subethamail.smtp.server.SMTPServer - SMTP server *:25 starting
16:12:18.937 [Test worker] INFO  o.subethamail.smtp.server.SMTPServer - SMTP server *:25 stopping
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
